apps/website/.vuepress/theme/global-components/CdsDemo.vue

```diff
@@ -45,6 +45,7 @@
 
 <script>
 import Prism from 'prismjs';
+import { html } from 'js-beautify';
```

Comment:
I tried several ways to format the code snippets in order to make the code formatting the same as the code snippets in our `angular-component` section. After 2-3 days of a struggle trying to make this work without a package, I decided to give it a try with a package and the package seems to help with our case.